scripts/build/repo.py

```diff
@@ -3,21 +3,36 @@
 """
 Build the package repository.
 """
-
-from datetime import datetime
-import gzip
-from enum import Enum, auto
 import logging
 import os
+import pathlib
 import shutil
-import textwrap
-from typing import Dict, Iterable, List, Optional, Set
+
+from datetime import datetime
+from enum import auto
+from enum import Enum
```

Comment:
```
from enum import auto, Enum
```

Reply:
That makes diffs have more noise when changing it in the future. It's also not being flagged as an issue by our linting, so it's not really something we care about in the reviews.